cogs/stats_commands.py

```diff
@@ -43,10 +45,16 @@ async def getStats(self, ctx, player_name):
         await ctx.send(embed=embed)
 
     @commands.command()
-    async def getMatchdayResults(self, ctx, matchday, season):
-        results = get_matchday_results(matchday, season)
-        embed = embedded_matchday_results(matchday, results)
-        await ctx.send(embed=embed)
+    async def matchday(
+        self,
+        ctx,
+        matchday=get_default_matchday("matchday"),
+        season=get_default_matchday("season"),
```

Comment:
I feel the naming of this method is a little confusing.  Since you can get both matchday or season default values it may make more sense to just call it `get_default` (or something like that) and pass in the string like you are now. I think that would make it more readable since it isn't specifically for matchdays.